types/profiles/v1/doc.go

```diff
@@ -20,4 +20,6 @@ limitations under the License.
 // Profiles are meant to be used by storage admins to communicate
 // their volume requirements to Maya without being involved into providing
 // exact values or numbers.
+
+// Package v1 creates documents file.
```

Comment:
remove this line. This package performs more functions. We don't need package comment. An empty line after line 22 will suffice.